src/containers/Header/Menus/LoggedIn/index.js

```diff
@@ -2,54 +2,50 @@ import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router';
 
-import { logout } from 'redux/modules/auth';
-import Button from 'components/Button';
+import headerStyles from '../../styles/index.scss';
 
 class LoggedInMenu extends Component {
   static propTypes = {
-    logout: PropTypes.func.isRequired,
     user: PropTypes.object,
   };
 
-  constructor(props) {
-    super(props);
-
-    this.logout = this.handleLogout.bind(this);
-  }
-
-  /**
-   * Handle logout
-   *
-   * @param {Event} event
-   */
-  handleLogout(event) {
-    event.preventDefault();
-    this.props.logout();
-  }
-
   render() {
     const { user } = this.props;
 
     if (!user) return null;
 
     return (
-      <nav>
-        <ul>
-          <li>
-            <Link to="/groups">Grupos</Link>
+      <div className={headerStyles.menuContent}>
+        <ul className={headerStyles.navigationList}>
+          <li className={headerStyles.navItem}>
+            <Link
+              to="/discover"
```

Comment:
Nothing I was trying with multiple menus. In the future here we could put the search functionality with a beautiful map full of groups :)